test/test-event-emitter.js

```diff
@@ -63,7 +63,7 @@ describe('widl-nan Unit Test - IDL EventEmitter', function() {
     });
   });
 
-  it('Return type of stop() is a Promise', done => {
+  it('Return type of stop() is a Promise', () => {
```

Comment:
When running this test case as single test-suite by `mocha` command, it will timeout.